Home/Home 3.2/TriangleTests.swift

```diff
@@ -4,85 +4,111 @@ class TriangleTests: XCTestCase {
     
     private let doubleAccuracy = 1e-9
     
-    func testArea_WithSidesThreeFourFive_ReturnsSix() {
+    func testCalculateArea_CorrectlyCalculatesArea_RightAngledTriangle() {
         // Arrange
-        let triangle = Triangle(a: 3, b: 4, c: 5)
+        let triangle = try! Triangle(a: 3, b: 4, c: 5)
         
         // Act
-        let result = triangle.area()
+        let result = triangle.calculateArea()
         
         // Assert
         XCTAssertEqual(result, 6.0, accuracy: doubleAccuracy)
     }
     
-    func testArea_WithSidesSixEightTen_ReturnsTwentyFour() {
+    func testCalculateArea_CorrectlyCalculatesArea_EquilateralTriangle() {
         // Arrange
-        let triangle = Triangle(a: 6, b: 8, c: 10)
+        let triangle = try! Triangle(a: 5, b: 5, c: 5)
         
         // Act
-        let result = triangle.area()
+        let result = triangle.calculateArea()
         
         // Assert
-        XCTAssertEqual(result, 24.0, accuracy: doubleAccuracy)
+        let expectedArea = (sqrt(3) / 4) * 25 // (sqrt(3) / 4) * a^2
```

Comment:
We can just calculate the value - it doesn't makes sense to have formula here - internal implementation can be changed but we expect that for given points some set of input produces some output.
Also formula may confuse readers.